README.md

```diff
@@ -31,16 +33,17 @@ npm install
 ```
 
 ### Development with Electron
-* Run `ng serve` to start the dev server.
-* Run `npm run start:electron:dev -- -testnet` in another window to start the electron application.
+* Run `npm run start:electron:dev -- -testnet` to start the electron application. Daemon will be updated and launched automatically.
```

Comment:
still needs `ng serve` no?

Reply:
I just tried `$ npm run start:electron:fast -- -testnet -debug` and seems like it's not needed anymore (or maybe rather included in the ^)

Reply:
That's the fast version, not the dev one tho

Reply:
Yeah true story.. Dev and fast are different.. No live reload with fast